hazelcast/src/main/java/com/hazelcast/instance/impl/HazelcastBootstrap.java

```diff
@@ -170,70 +170,60 @@ public static synchronized void executeJar(@Nonnull Supplier<HazelcastInstance>
             }
 
         } finally {
-            // HazelcastInstance should be closed if called by client side
-            // HazelcastInstance should not be closed if called by member side
+            // HazelcastInstance should be closed only if called by the client side
             if (!calledByMember) {
                 HazelcastInstance remembered = HazelcastBootstrap.supplier.remembered();
                 if (remembered != null) {
                     try {
                         remembered.shutdown();
-                    } catch (Throwable t) {
+                    } catch (Exception exception) {
```

Comment:
Why we stop to catch Throwable here? Related commit message says "Changes for review comments".

Reply:
Throwable is too generic. Then code is catching Error too. But javadoc from Error is 
```
An {@code Error} is a subclass of {@code Throwable}
that indicates serious problems that a reasonable application
should not try to catch. Most such errors are abnormal conditions.
```